dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassFile.java

```diff
@@ -28,29 +28,35 @@
  */
 public final class ClassFile {
   private final Path jar;
-  private final String className;
+  private final String binaryName;
 
+  @VisibleForTesting
   public ClassFile(Path jar, String className) {
     this.jar = checkNotNull(jar);
-    this.className = checkNotNull(className);
+    this.binaryName = checkNotNull(className);
   }
 
   /** Returns the path to the JAR file containing the class. */
   public Path getJar() {
     return jar;
   }
 
-  /** Returns class name (binary name as in {@link Symbol#getClassName()}) */
-  public String getClassName() {
-    return className;
+  /**
+   * Returns the binary name of the class as defined by the 
+   * <a href="https://docs.oracle.com/javase/specs/jls/se8/html/jls-13.html#jls-13.1">Java
+   * Language Specification</a> and
+   * found in the .class files.
```

Comment:
```suggestion
   * found in the .class file.
```

Or this doesn't need "and found in the .class file" part.

Reply:
done